File: Pokemon/Models/Pokemon.cs
using MySql.Data.MySqlClient;
using Dapper.Contrib.Extensions;

namespace Pokemon
{
    public class Pokemon
    {
        public int id { get; set; }
        public string name { get; set; }
        public int height { get; set; }
        public bool is_default { get; set; } 
        public int order { get; set; } 
        public int weight { get; set; }
        public List<Move> moves { get; set; }
        public PokemonSprites sprites { get; set; }
        public SpeciesName species { get; set; }
        public List<Stats> stats { get; set; }
        public List<Type> types { get; set; }
    }
}

File: PokemonFront/src/app/pokemon.ts
import { Move } from "./move";
import { PokemonSprites } from "./pokemon-sprites";
import { SpeciesName } from "./species-name";
import { Stats } from "./stats";
import { Type } from "./type";

export interface Pokemon {
    id: number,
    name: string,
    height: number,
    is_default: boolean,
    order: number,
    weight: number,
    moves: Move[],
    sprites: PokemonSprites,
    species: SpeciesName,
    stats: Stats[], 
    types: Type[]
}
